test/test_datasets.py

```diff
@@ -110,5 +111,67 @@ def test_yesno(self):
             assert label == expected_label
 
 
+class TestLibriTTS(TempDirMixin, TorchaudioTestCase):
+    backend = 'default'
+
+    root_dir = None
+    data = []
+    utterance_ids = [
+        [19, 198, '000000', '000000'],
+        [26, 495, '000004', '000000'],
+    ]
+    original_text = 'this is the original text.'
+    normalized_text = 'this is the normalized text.'
+
+    @classmethod
+    def setUpClass(cls):
+        cls.root_dir = cls.get_base_temp_dir()
+        base_dir = os.path.join(cls.root_dir, 'LibriTTS', 'train-clean-100')
+        for utterance_id in cls.utterance_ids:
+            filename = f'{"_".join(str(u) for u in utterance_id)}.wav'
+            file_dir = os.path.join(base_dir, str(utterance_id[0]), str(utterance_id[1]))
+            os.makedirs(file_dir, exist_ok=True)
+            path = os.path.join(file_dir, filename)
+
+            data = get_whitenoise(sample_rate=8000, duration=6, n_channels=1, dtype='int16')
```

Comment:
I just realized that I forgot to add `seed` to `get_whitenoise` function call in #792 and all the data generated are identical. Can you add `seed`?

```
for i, utterance_id in enumerate(cls.utterance_ids):
    ...
    data = get_whitenosie(..., seed=i)
```

Reply:
The seed has been added.